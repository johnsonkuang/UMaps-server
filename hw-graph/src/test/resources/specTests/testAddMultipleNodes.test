# Testing Add Multiple Nodes
# Tests that multiple nodes can be added into a created graph

# Create a graph
CreateGraph graph1

# Add three nodes to the graph
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3