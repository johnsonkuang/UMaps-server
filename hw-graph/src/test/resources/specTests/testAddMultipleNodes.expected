# Testing Add Multiple Nodes
# Tests that multiple nodes can be added into a created graph

# Create a graph
created graph graph1

# Add three nodes to the graph
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1